python/pyarrow/tests/test_compute.py

```diff
@@ -2111,6 +2122,7 @@ def test_round_temporal(unit):
         "1967-02-26 05:56:46.922376960",
         "1975-11-01 10:55:37.016146432",
         "1982-01-21 18:43:44.517366784",
+        "1992-01-01 00:00:00.100000000",
```

Comment:
Can we add a dedicated test for `multiple_since_greater_unit`?

Reply:
Added, but not aesthetically pleasing :)